src/components/data-exploration/cell-sets-tool/CellSetsTool.jsx

```diff
@@ -124,7 +129,7 @@ const CellSetsTool = (props) => {
   const renderContent = () => {
     let operations = null;
 
-    if (numSelectedCellSetKeys > 0) {
+    if (numSelectedCellSetKeys !== null) {
```

Comment:
Before, if you selected a cell class that was empty it wouldn't say "0 cells selected" because `numSelectedCellSetKeys === 0` meant that nothing was shown.

I added null so that:
1. nothing is selected ---> `numSelectedCellSetKeys = null` 
2. the are selected things but they have 0 cells ---> `numSelectedCellSetKeys = 0`

In this way, we can show `"0 cells selected"` in case 2.